website/gatsby-node.js

```diff
@@ -26,26 +26,7 @@ exports.onCreateWebpackConfig = ({ actions, stage }) => {
     };
   }
 
-  actions.setWebpackConfig({
-    resolve: {
-      alias,
-    },
-    /**
-     * Inspiration from https://github.com/gatsbyjs/gatsby/issues/10965. There seems to be an issue where a couple dependecies are duplicated
-     * across each page component and not properly split out into commons chunks (prettier, babel parser are the main ones)
-     */
-    optimization: {
```

Comment:
This optimization is no longer needed in new version

Reply:
Love the cleanup!